src/explore-education-statistics-common/src/components/ErrorSummary.tsx

```diff
@@ -43,7 +54,11 @@ export const BaseErrorSummary = forwardRef<
       data-testid={testId}
     >
       <div role="alert">
-        <h2 className="govuk-error-summary__title">{title}</h2>
+        {createElement(
```

Comment:
Why the need for createElement here, just out of interest?

Reply:
Because you can now specify what heading tag to use, instead of always h2, so it needs to create the element using that.